src/Web/Controllers/BasketController.cs

```diff
@@ -401,6 +401,22 @@ private Purchase SetupPurchase(CheckoutDetailsViewModel checkoutDetails, BasketV
                 BillingAddress = billingAddress,
             };
 
+            Product mostExpensiveProduct = productList.FirstOrDefault();
```

Comment:
Could simplify

```var mostExpensiveProduct = productList.Max(p => p.SalesPrice / p.Quantity);```

Reply:
tried that, but that gives sales price of the most expensive product.  wanted to include the name of the product so we could test sending a string value in custom data.